modules/core/src/main/java/org/apache/ignite/internal/visor/query/VisorQueryCancelTask.java

```diff
@@ -17,22 +17,25 @@
 
 package org.apache.ignite.internal.visor.query;
 
+import java.util.Collection;
```

Comment:
Imports

Reply:
What's wrong with it?

Reply:
Unused imports. At least my IDE thinks so.

Reply:
it used in the method `jobNodes`:

```
    @Override protected Collection<UUID> jobNodes(VisorTaskArgument<VisorQueryCancelTaskArg> arg) {
        return Collections.singleton(arg.getArgument().getNodeId());
    }
```

Reply:
Ok, might be I missed it. You are right.